.vib/spring-cloud-dataflow/cypress/cypress/support/commands.js

```diff
@@ -0,0 +1,26 @@
+const COMMAND_DELAY = 800;
+const TYPE_DELAY = 200;
```

Comment:
Could you please give a bit of context regarding the need for a `TYPE_DELAY`? 

Looking now at the const names, shouldn't we be using `CLICK_DELAY` instead of `COMMAND_DELAY`?

Reply:
Yes, you are right. I fixed this mistake.
The context: when tests run in sequence sometimes Cypress types the values too quickly and one of the letters (usually the first one) doesn't get typed in properly. 
This flakiness is solved by adding a small delay in typing.